portfolio/src/main/java/com/google/sps/servlets/AuthCheckServlet.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.sps.data.User;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/auth/check")
+public class AuthCheckServlet extends HttpServlet {
+
+  @Override
+  protected void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws ServletException, IOException {
+    UserService userService = UserServiceFactory.getUserService();
+
+    response.setContentType("application/json");
+
+    if (!userService.isUserLoggedIn()) {
+      StringBuilder loggedOutResp = new StringBuilder();
+      loggedOutResp.append("{");
```

Comment:
Could you use a json library instead of writing it manually?

Reply:
I was basing this off of the examples, but I have found a much better way of doing this. Will use that instead and can delete the User class which was created just so I could use the Gson library.